universities/organizations.json

```diff
@@ -861,6 +861,18 @@
       "latitude": 50.87959
     }
   },
+  {
+    "id": "11352",
+    "icpc_id": "11352",
+    "name": "RUB",
+    "formal_name": "Ruhr University",
```

Comment:
For the "kick up the totem pole list": It looks like their official name includes "Bochum" :stuck_out_tongue: 

- https://www.ruhr-uni-bochum.de/en
- https://en.wikipedia.org/wiki/Ruhr_University_Bochum

Cool to see a new name! :smile: